Control/Metrics/src/ArmoniK.Core.Control.Metrics.csproj

```diff
@@ -44,5 +44,8 @@
     <ProjectReference Include="..\..\..\Adaptors\Redis\src\ArmoniK.Core.Adapters.Redis.csproj" />
     <ProjectReference Include="..\..\..\Common\src\ArmoniK.Core.Common.csproj" />
   </ItemGroup>
-
+	
+  <ItemGroup>
+	<AssemblyAttribute Include="System.Diagnostics.CodeAnalysis.ExcludeFromCodeCoverageAttribute" />
```

Comment:
Why is this (non-test) project is excluded from coverage?

Reply:
This is a helper project to collect metrics, I don't think its part of the core functionalities of the library so it should not be tested by us.